lib/event_planner.rb

```diff
@@ -0,0 +1,161 @@
+require "./games"
+
+class EventPlanner
```

Comment:
Really like all the work that went into this. The `ellipse` method is a nice touch. Code here seems pretty clean. Like that you've wrapped this in a class and kept your runner skinny.